### DISCUSSION TITLE
Open a extra port beyond 80 and 443 for container!

### DISCUSSION THREAD

**Discussion by marciliojunior** [nginx-proxy/nginx-proxy#2476] [Q&A]
I'm using nginx-proxy successfully for containers involving ports 80 and 443. However, my project also uses port 6001 (for websocket connection). The connection via this port 6001 is not working (neither via ws:// nor wss://). How do I enable access via wss://?

**Comment by buchdag** [comment]
Hi @marciliojunior. nginx-proxy does not listen on new ports beside 80 and 443 depending on protocols and there is no specific built in support for Websocket (I'm not really familiar with reverse proxying Websocket by the way).

Could you describe your project in more details and how nginx-proxy should ideally work with it ?

**Reply by marciliojunior** [reply]
Hi @buchdag. Thanks for the reply. 

My project involves the following:
Currently I use the nginxproxy/nginx-proxy project together with the nginxproxy/acme-companion project (for automatic management of ssl certificates). Everything works perfectly and my containers can be accessed via "https" normally. However, my container, in addition to being accessed through ports 80/433, is also exposed to port 6001 for access via the websockets protocol. Access through "ws://" works normally (the port is exposed in the docker-compose of the informed container).

My goal would be to access port 6001 using "wss://". In other words, use the ssl certificate generated through the nginxproxy/acme-companion container for websocket access (in the same way as it is done through port 433). When I try to access my project through "wss://" the request doesn't even reach the container.

I hope I made my objective clear (sorry for the limited English).
